mantaray/backend.py

```diff
@@ -502,7 +502,21 @@ def send_privmsg(self, nick_or_channel: str, text: str) -> None:
             done_event=SentPrivmsg(nick_or_channel, text),
         )
 
+
+    def kick(self, channel: str, nick: str, reason: str | None = None) -> None:
+        if reason is None:
+            reason=''
+            
+        print(channel, nick, reason)
+        self._send_soon(
+            "KICK",
+            channel,
+            nick,
+            " Reason:" + reason
+            )
+
     # emits SelfChangedNick event on success
+            
```

Comment:
Delete the extra space. No reason to add it.

Reply:
Deleted